### PR TITLE
connect: Add support for notarization

### PR DESCRIPTION
**Pull request by zmb3**
Notarization is performed via `electron-notarize` when the
`APPLE_USERNAME` and `APPLE_PASSWORD` environment variables are set.

### CODE REVIEW

**Review by ravicious — APPROVED** [review]
Obviously I didn't run notarize, but I did execute `yarn package-term`. Looks good to me once we take care of that yarn.lock issue.

File: yarn.lock [hunk 1]
```diff
@@ -3336,11 +3336,6 @@ abab@^2.0.3, abab@^2.0.5:
   resolved "https://registry.yarnpkg.com/abab/-/abab-2.0.5.tgz#c0b678fb32d60fc1219c784d6a826fe385aeb79a"
   integrity sha512-9IK9EadsbHo6jLWIpxpR6pL0sazTXV6+SQv25ZB+F7Bj9mJNaOc4nCRabwd5M/JwmUa8idz6Eci6eKfJryPs6Q==
 
-abbrev@1:
```
[the remark this answers is not in the document]
Comment (zmb3) [reply yarn.lock]: I wonder if it was my node version. Tried again with node 16 - see what happens now.

[the remark this answers is not in the document]
Comment (ravicious) [reply yarn.lock]: Looks good now!

**Review by gzdunek — APPROVED** [review]
Looks good!

**Pull request merged**